tests/test_timestamp_formats.py

```diff
@@ -0,0 +1,67 @@
+from pprint import pprint
+import pytest
+
+from datetime import datetime, timezone, timedelta
+from logmerger.timestamp_wrapper import TimestampedLineTransformer
+
+local_time = datetime.now().astimezone()
+local_tz = local_time.tzinfo
+
+def _test_timestamp_format_parsing(string_date, expected_datetime):
```

Comment:
Nit: please add type annotations to the method signatures. Or check in as-is, and I can add them afterward.

Reply:
Sure thing. I added the types here.